recipes/gcc/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "4.8.5" %}
+
+package:
+  name: gcc
+  version: {{ version }}
+
+source:
+  fn: gcc-{{ version }}.tar.bz2
+  url: http://www.netgull.com/gcc/releases/gcc-{{ version }}/gcc-{{ version }}.tar.bz2
+  md5: 80d2c2982a3392bb0b89673ff136e223
+
+build:
+  skip: true                             # [win]
+  detect_binary_files_with_prefix: true  # [not linux32]
+  number: 5
+
+requirements:
+  build:
+    # These are taken from the output of the configure scripts
+    - gmp >=4.3.2
+    - mpfr >=2.4.2
+    - mpc >=0.8.1
+    - isl >=0.10,<=0.14
+    - cloog 0.18.0
+    # Do not make gcc a build dependency (you will need to add it to the PATH manually)
```

Comment:
We have to build the compiler somehow. The comment is merely reminding people to not to add the packaged `gcc` to the build environment to do so if they chose to use the packaged `gcc`. The concern here is that when the install occurs we may get stuck with some pieces from the old compiler in our final package. This almost certainly will cause horrible errors from the newly packaged `gcc` that are showstoppers that are easily recognizable or potentially weird unclear errors. This is likely just a warning to avoid bad behavior as this could be used by anyone, anywhere, anyway and so has few guarantees.

My understanding is that Continuum built this using the standard system `gcc` on CentOS 5 in either a minimal VM or a Docker container. Their Mac builds I am less clear on. FWICT it appears these were built with the system `gcc` or Continuum's `gcc` and then re-linked with `install_name_tool` to use the libraries included with the compiler.